projects/kit/components/input-phone-international/input-phone-international.component.ts

```diff
@@ -51,13 +53,16 @@ import {from, skip} from 'rxjs';
 
 import {TuiGetCountryCallingCodePipe} from './get-country-calling-code.pipe';
 import {TUI_INPUT_PHONE_INTERNATIONAL_OPTIONS} from './input-phone-international.options';
+import {TuiAutoFocus, tuiFilterList} from '@taiga-ui/cdk';
```

Comment:
Imports should be alphabetically sorted by precommit, looks like something went wrong or you skipped it, please move them to place above. Also it should have changed it to deep imports:
```ts
import {TuiAutoFocus} from '@taiga-ui/cdk/directives/auto-focus';
```